agents-for-bedrock/features-examples/11-create-agent-with-code-interpreter/files/SPY.csv

```diff
@@ -0,0 +1,254 @@
+Symbol,Date,Open,High,Low,Close,Adj Close,Volume

```

Comment:
need to make reference to data source and preferably download the data from the source on the notebook